spec/models/document_spec.rb

```diff
@@ -1,5 +1,49 @@
 require 'rails_helper'
 
 RSpec.describe Document, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it "should have a valid factory girl" do
```

Comment:
Remove 'should', use something like `it "has a valid factory girl"`
http://www.betterspecs.org/#expect